daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -1096,6 +1096,14 @@ func (devices *DeviceSet) initDevmapper(doInit bool) error {
 		}
 	}
 
+	if devices.thinPoolDevice == "" {
```

Comment:
This is right below the closing brace of `if info.Exists == 0 && devices.thinPoolDevice == "" {`, there's probably some way to make this more readable.